### DISCUSSION TITLE
Modem LTE stick E3272 HiLink failed

### DISCUSSION THREAD

**Discussion by seb14173** [tesla-android/issue-tracker#256] [Q&A]
Hello everyone, I bought an old e3272 Huawei for android tesla internet connection to replace my usb tethering from my phone. I removed the pin code from the sim card and the stick works well under windows (4g internet access) but as soon as you plug it into the raspberry the testa-android hotspot disappears ? Has anyone had this problem ? thanks

**Comment by mikegapinski** [comment]
If the hotspot disappears this means the system died. Check the power source and cable. If the pi does not get enough amperage it crashes when usb devices are connected.

What is the variant of the modem? e3272-xxx

**Reply by seb14173** [reply]
e3272 variant is e3272s-153. I have replaced power supply but no change. I need reboot to access Tesla-Android hotspot.
